Sources/BraintreePayPalNativeCheckout/BTPayPalNativeOrderCreationClient.swift

```diff
@@ -33,13 +33,11 @@ class BTPayPalNativeOrderCreationClient {
             }
 
             guard let paypalEnabled = config.json?["paypalEnabled"].isTrue, paypalEnabled else {
-                self.apiClient.sendAnalyticsEvent("ios.paypal-native.create-order.paypal-not-enabled.failed")
```

Comment:
`tokenize:failed` from these events are handled in `createOrder` completion block in `BTPayPalNativeCheckoutClien`t above.